### DISCUSSION TITLE
The compile_commands.json file has `-std=gnu11` added to the command, but still pops up a warning about the GNU extension syntax.

### DISCUSSION THREAD

**Discussion by wc7086** [clangd/clangd#2035] [Q&A] [answered]
I'm wondering if this was a bug or expected behavior?

![image](https://github.com/clangd/clangd/assets/74053663/3a95b9b4-81a8-489b-a263-5aa32c54e857)

**Comment by HighCommander4** [comment, marked as answer]
I only get this diagnostic if my compile command explicitly contains the option `-Wgnu-zero-variadic-macro-arguments `. In that case, it's expected, regardless of `-std=gnu11`.

**Comment by wc7086** [comment]
https://discourse.llvm.org/t/next-batch-of-wgnu-flags/28852
